app/models/middleware_datasource.rb

```diff
@@ -3,4 +3,10 @@ class MiddlewareDatasource < ApplicationRecord
   belongs_to :middleware_server, :foreign_key => "server_id"
   acts_as_miq_taggable
   serialize :properties
+
+  include LiveMetricsMixin
+
+  def metrics_capture
+    @metrics_capture ||= ManageIQ::Providers::Hawkular::MiddlewareManager::LiveMetricsCapture.new(self)
```

Comment:
does this mean that for every datasource object/instance in miq, a new instance of livemetrics capture is created?
(I'm wondering efficiency/performance wise)